### PR TITLE
docs: example usage in readme

### PR DESCRIPTION
**Pull request by ab93**
Signed-off-by: Avik Basu <avikbasu93@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,30 +38,146 @@ For an always-on ML platform, the key requirement is the ability to train or ret
...
+Now that the model is trained, let's save it. Numalogic has built in support 
+for Mlflow's tracking and logging system.
+```python
+from numalogic.registry import MLflowRegistrar
+
+# static and dynamic keys are used to look up a model
+static_keys = ["synthetic", "3ts"]
+dynamic_keys = ["minmaxscaler", "sparseconv1d"]
+
+registry = MLflowRegistrar(tracking_uri="http://0.0.0.0:8080", artifact_type="pytorch")
```
Comment (vigith) [inline README.md:140]: we never documented how to start MLFlow server. may be we should add that too in `examples/` folder? Else the readme will not be complete.

Comment (ab93) [reply README.md:140]: Yes, that's fair. We don't have an `examples` folder yet. But we can add one.

Comment (s0nicboOm) [reply README.md:140]: Here is an elaborate link that we should point community to for this: https://www.mlflow.org/docs/latest/tracking.html#scenario-1-mlflow-on-localhost

Comment (ab93) [reply README.md]: Referenced that link

**Review by s0nicboOm — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (vigith) [reply README.md]: @s0nicboOm, I do not see any installation steps in that link.

File: README.md [hunk 2]
```diff
@@ -130,14 +130,21 @@ model.fit(X_train)
 ### Model saving
 Now that the model is trained, let's save it. Numalogic has built in support 
 for Mlflow's tracking and logging system.
+
+Let's first start the [mlflow server on localhost](https://www.mlflow.org/docs/latest/tracking.html#scenario-1-mlflow-on-localhost):
```
Comment (vigith) [inline README.md:134]:
```suggestion
Let's first start the [mlflow server on localhost](https://www.mlflow.org/docs/latest/tracking.html#scenario-1-mlflow-on-localhost), which has already been installed via `poetry` dependency:
```

**Review by vigith — APPROVED** [review]

**Pull request merged**